src/libs/Navigation/Navigation.js

```diff
@@ -43,7 +43,7 @@ function goBack() {
  * @param {String} route
  */
 function navigate(route = ROUTES.HOME) {
-    if (route === ROUTES.HOME) {
+    if (route === ROUTES.HOME || route === ROUTES.WORKSPACE) {
```

Comment:
NAB, when we are navigating to `/workspace` we are assuming that we're already navigated to `/workspace/card` or some other route. We should maybe add a comment to clarify what's happening here and the intended behavior e.g.

>If someone tries to navigate to `/workspace` when the workspace drawer is not open the home drawer will toggle. In this case, we should navigate to a sub-route of workspace instead e.g. `/workspace/card`

Reply:
Actually, this also kind of makes me think that it might be incorrect to have this check in general right?

We already established that calling `Navigation.goBack()` will toggle the drawer. So maybe we can do that instead of navigating to `/workspace` when we want to toggle the workspace sidebar open?

Reply:
But when we will navigate over the WorkspaceSetting Modal, then go back will switch those screens, Let's say first we got Card page then people page. So on the people page, goback should take us to Card apge.

Reply:
@marcaaron Please tell me if I need to do these changes as well. Otherwise, PR is ready for review.

Reply:
I'm not really following as there's no way to access the the `/workspace/people` from `/workspace/card`, but it's not a blocker for me just something that seems weird / confusing.

Reply:
Ok. I think it's better than we can adjust that later when we implement other pages like /workspace/people.